documentation/00_changelog.md

```diff
@@ -0,0 +1,92 @@
+---
+id: changelog
+title: Changelog
+sidebar_label: Changelog
+---
+
+:::info
+The Leo language and Aleo SDK are currently in active development. Please monitor their respective repositories for 
+the latest updates:
+[**Leo**]
+(https://github.com/AleoHQ/leo) [**Aleo**]
+(https://github.com/AleoHQ/aleo) .
+:::
+
+## Changes **06.14.2023**
+---
+### Hash Functions
```

Comment:
Can you specify that these opcodes are for Aleo Instructions and provide links to the full documentation pages?

Reply:
Yes, will do!